modules/rational-code-completion.el

```diff
@@ -0,0 +1,72 @@
+;;; rational-code-completion.el -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022
+;; SPDX-License-Identifier: MIT
+
+;; Author: System Crafters Community
+
+;; Commentary
+
+;; Setup code completion packages. Code completion is done with corfu
+;; rather than the traditional company to better utilize native
+;; completion features provided by emacs.
+
+;;; Code:
+
+(straight-use-package 'corfu)
+(straight-use-package 'cape)
+(straight-use-package 'orderless)
+
+(require 'corfu)
+
+(defcustom ycm-style nil
```

Comment:
This should be defined as `rational-code-completion-ycm-style` to stay consistent with the other modules.

Reply:
Nice save, forgot to change that. On it now

Reply:
This is resolved in the most recent commit